examples/06_command_framework/src/main.rs

```diff
@@ -88,10 +88,11 @@ fn main() {
         .command("commands", |c| c
             .check(owner_check)
             .exec(commands))
-        .command("emoji cat", |c| c
-            .exec_str(":cat:")
-            .required_permissions(permissions::SEND_MESSAGES))
-        .command("emoji dog", |c| c.exec_str(":dog:"))
+        .group("emoji", |g|
+            .command("cat", |c| c
+                .exec_str(":cat:")
+                .required_permissions(permissions::SEND_MESSAGES))
```

Comment:
It was probably a mistake of mine to use this permission as an example previously; maybe `ADMINISTRATOR`?